zerver/webhooks/patreon/fixtures/members_create.json

```diff
@@ -0,0 +1,384 @@
+        "twitter": "zulip",
+        "url": "https://www.patreon.com/zulip",
+        "vanity": "zulip",
+        "youtube": null
+      },
+      "id": "7498102",
+      "relationships": {
+        "campaign": {
+          "data": {
+            "id": "1194966",
+            "type": "campaign"
+          },
+          "links": {
+            "related": "https://www.patreon.com/api/campaigns/1194966"
+          }
+        }
+      },
+      "type": "user"
+    },
+    {
+      "attributes": {
+        "amount": 0,
+        "amount_cents": 0,
+        "created_at": null,
+        "description": "Everyone",
+        "patron_currency": "USD",
+        "remaining": 0,
+        "requires_shipping": false,
+        "url": null,
+        "user_limit": null
+      },
+      "id": "-1",
+      "type": "reward"
+    },
+    {
+      "attributes": {
+        "amount": 1,
+        "amount_cents": 1,
+        "created_at": null,
+        "description": "Patrons Only",
+        "patron_currency": "USD",
+        "remaining": 0,
+        "requires_shipping": false,
+        "url": null,
+        "user_limit": null
+      },
+      "id": "0",
+      "type": "reward"
+    },
+    {
+      "attributes": {
+        "amount": 500,
+        "amount_cents": 500,
+        "created_at": "2021-04-07T22:29:15.334+00:00",
+        "currency": "USD",
+        "description": "Contribute to Zulip's development and free hosting for open source projects and other worthy organizations!",
+        "discord_role_ids": null,
+        "edited_at": "2021-04-07T22:29:15.334+00:00",
+        "image_url": null,
+        "patron_amount_cents": 500,
+        "patron_count": 3,
+        "patron_currency": "USD",
+        "post_count": 0,
+        "published": true,
+        "published_at": "2021-04-07T22:29:15.334+00:00",
+        "remaining": null,
+        "requires_shipping": false,
+        "title": "Backer",
+        "unpublished_at": null,
+        "url": "/join/zulip/checkout?rid=7027346",
+        "user_limit": null,
+        "welcome_message": null,
+        "welcome_message_unsafe": null,
+        "welcome_video_embed": null,
+        "welcome_video_url": null
+      },
+      "id": "7027346",
+      "relationships": {
+        "campaign": {
+          "data": {
+            "id": "1194966",
+            "type": "campaign"
+          },
+          "links": {
+            "related": "https://www.patreon.com/api/campaigns/1194966"
+          }
+        }
+      },
+      "type": "reward"
+    },
+    {
+      "attributes": {
+        "amount": 1000,
+        "amount_cents": 1000,
+        "created_at": "2021-04-07T22:32:19.565+00:00",
+        "currency": "USD",
+        "description": "Contribute to Zulip's development and free hosting for open source projects and other worthy organizations!",
+        "discord_role_ids": null,
+        "edited_at": "2021-04-07T22:32:19.565+00:00",
+        "image_url": null,
+        "patron_amount_cents": 1000,
+        "patron_count": 1,
```

Comment:
See, more than one value for `patron_count`

Reply:
I believe the one mentioned here is the `patron_count` for that particular tier. The `patron_count` within `data["attributes"]` might be the total count, but I'm unable to find a reference in their documentation to link to.